CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Features
 
 ### Fixes
+- Fix counting false negatives and false positives in table structure evaluation
```

Comment:
You need to bump the version to `0.14.10-dev0` and run `make version-sync`. This way  you should have 2 files to commit - `CHANGELOG.md` and `unstructured/__version__.py`